Traceback (most recent call last):
  File "C:\Users\Avery\anaconda3\envs\JupyterBook\lib\site-packages\jupyter_cache\executors\utils.py", line 51, in single_nb_execution
    executenb(
  File "C:\Users\Avery\anaconda3\envs\JupyterBook\lib\site-packages\nbclient\client.py", line 1117, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "C:\Users\Avery\anaconda3\envs\JupyterBook\lib\site-packages\nbclient\util.py", line 78, in wrapped
    return just_run(coro(*args, **kwargs))
  File "C:\Users\Avery\anaconda3\envs\JupyterBook\lib\site-packages\nbclient\util.py", line 57, in just_run
    return loop.run_until_complete(coro)
  File "C:\Users\Avery\anaconda3\envs\JupyterBook\lib\asyncio\base_events.py", line 642, in run_until_complete
    return future.result()
  File "C:\Users\Avery\anaconda3\envs\JupyterBook\lib\site-packages\nbclient\client.py", line 558, in async_execute
    await self.async_execute_cell(
  File "C:\Users\Avery\anaconda3\envs\JupyterBook\lib\site-packages\nbclient\client.py", line 862, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "C:\Users\Avery\anaconda3\envs\JupyterBook\lib\site-packages\nbclient\client.py", line 765, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
import numpy as np
import pandas as pd

np.random.seed(24)
df = pd.DataFrame({'A': np.linspace(1, 10, 10)})
df = pd.concat([df, pd.DataFrame(np.random.randn(10, 4), columns=list('BCDE'))],
               axis=1)
df.iloc[3, 3] = np.nan
df.iloc[0, 2] = np.nan

def color_negative_red(val):
    """
    Takes a scalar and returns a string with
    the css property `'color: red'` for negative
    strings, black otherwise.
    """
    color = 'red' if val < 0 else 'black'
    return 'color: %s' % color

def highlight_max(s):
    '''
    highlight the maximum in a Series yellow.
    '''
    is_max = s == s.max()
    return ['background-color: yellow' if v else '' for v in is_max]

df.style.\
    applymap(color_negative_red).\
    apply(highlight_max).\
    set_table_attributes('style="font-size: 10px"')
------------------

[1;31m---------------------------------------------------------------------------[0m
[1;31mModuleNotFoundError[0m                       Traceback (most recent call last)
[1;32m~\AppData\Local\Temp/ipykernel_12644/2549200461.py[0m in [0;36m<module>[1;34m[0m
[0;32m      1[0m [1;32mimport[0m [0mnumpy[0m [1;32mas[0m [0mnp[0m[1;33m[0m[1;33m[0m[0m
[1;32m----> 2[1;33m [1;32mimport[0m [0mpandas[0m [1;32mas[0m [0mpd[0m[1;33m[0m[1;33m[0m[0m
[0m[0;32m      3[0m [1;33m[0m[0m
[0;32m      4[0m [0mnp[0m[1;33m.[0m[0mrandom[0m[1;33m.[0m[0mseed[0m[1;33m([0m[1;36m24[0m[1;33m)[0m[1;33m[0m[1;33m[0m[0m
[0;32m      5[0m [0mdf[0m [1;33m=[0m [0mpd[0m[1;33m.[0m[0mDataFrame[0m[1;33m([0m[1;33m{[0m[1;34m'A'[0m[1;33m:[0m [0mnp[0m[1;33m.[0m[0mlinspace[0m[1;33m([0m[1;36m1[0m[1;33m,[0m [1;36m10[0m[1;33m,[0m [1;36m10[0m[1;33m)[0m[1;33m}[0m[1;33m)[0m[1;33m[0m[1;33m[0m[0m

[1;31mModuleNotFoundError[0m: No module named 'pandas'
ModuleNotFoundError: No module named 'pandas'

